### ISSUE TITLE
Auto scroll file with drag control

### ISSUE THREAD

**Issue by ronrdev** [artem-barmin/obsidian-block-drag-n-drop#11]
I'm not sure if it is possible to take control of the content view with the drag action. When dragging a block past the top or bottom of screen space - have file scroll up or down the document. 

![Screenshot_2022-05-01_21-31-03](https://user-images.githubusercontent.com/13022919/166174436-61d12186-01ee-4dc0-bfe1-859ba918a5b7.png)

**Comment by artem-barmin** [comment]
Accepted. Will add it to next release

**Comment by artem-barmin** [comment]
@ronrdev Hi. Just tried to reproduce, and look's like it already works:

https://user-images.githubusercontent.com/11332147/167298515-2dd5c303-511f-42e2-8624-c7cf84be9d21.mov

Please share your theme/plugins so I will try to reproduce it

**Comment by ronrdev** [comment]
Hi. Okay, I should have tested it more thoroughly. It works easy for me with the default theme. With minimal theme It is more difficult and feels slow to respond. I'm not sure what triggers the scroll. Looks like moving the cursor passed the top isn't enough, you have to hover at the edge. It's my configuration causing the trouble, not the plugin. 


https://user-images.githubusercontent.com/13022919/167307642-97f73a14-1e01-41ea-bff8-81300ad478dc.mp4




Thank you.

**Comment by artem-barmin** [comment]
Yes, please add bug report. I will think how to support Minimal theme
